packages/zero-runtime/README.md

```diff
@@ -0,0 +1,244 @@
+# zero-runtime
+
+A zero-runtime CSS-in-JS library that extracts the colocated css to it's own css files at build-time.
+
+## Getting started
+
+Zero-runtime supports Next.js and Vite—you must install the corresponding plugin, as shown below. The package currently has a dependency on `@mui/material` to initialize the theme object, but this is only at build time. There won't be any Material UI code at runtime if you're not using it otherwise—in that case, you can move it to dev dependencies instead (as shown with the plugins).
+
+### Installation
+
+```bash
+npm install @mui/zero-runtime @mui/material
+# For Next.js
+npm install --save-dev @mui/zero-next-plugin
+# For Vite
+npm install --save-dev @mui/zero-vite-plugin
+```
+
+#### Next.js
+
```

Comment:
```suggestion
```bash
npm install @mui/zero-runtime @mui/material @mui/zero-next-plugin
```
```

Reply:
`@mui/zero-next-plugin` is a dev dependency though.

Reply:
My point is it looks cleaner to move the installation command to each framework even though a few lines are duplicated.

Reply:
```suggestion
```bash
npm install @mui/zero-runtime @mui/material
npm install --save-dev @mui/zero-next-plugin
```
```